addons/website/static/src/xml/website.seo.xml

```diff
@@ -4,6 +4,13 @@
     <t t-name="website.editorbar.menu.promote">
         <li><a data-action="promote-current-page" href="#"><span title="Promote page on the web">Promote</span></a></li>
     </t>
+
+    <t t-name="Configurator.language_promot">
```

Comment:
It is template of selection box that contain a languages which are define on website.